definitions/features/katello.rb

```diff
@@ -14,4 +14,17 @@ def data_dirs
   def current_version
     @current_version ||= rpm_version('katello')
   end
+
+  def services
+    # TODO: remove foreman services, where should they go?
+    {
+      'postgresql'               => 5,
+      'qpidd'                    => 10,
+      'qdrouterd'                => 10,
+      'httpd'                    => 30,
+      # 'puppetserver'             => 30,
+      'foreman-tasks'            => 30,
+      'goferd'                   => 30
+    }
```

Comment:
@iNecas @mbacovsky what feature should base foreman services be stored in? (postgresql, httpd)

Reply:
ForemanServer for httpd and ForemanDatabase for postgresql should be good enough IMO